cg/services/deliver_files/delivery_file_fetcher_service/models.py

```diff
@@ -24,4 +24,6 @@ class SampleFile(BaseModel):
 class DeliveryFiles(BaseModel):
     delivery_data: DeliveryMetaData
     case_files: list[CaseFile] | None
-    sample_files: list[SampleFile]
+    sample_files: list[SampleFile] = Field(
+        ..., min_length=1, description="At least one sample file is required for delivery."
+    )
```

Comment:
I do not think that workflows will always have a sample file to deliver. What is the intended purpose of the validation error? To me, it would seem like this should only throw an error when we have asked it to deliver sample files, but there are no files to deliver. This logic would throw and error every time, even though we are not expecting any sample files to be delivered.

Reply:
discussed over slack - can become an issue in the future, right now all cases have some sort of sample files attached to it. (bam/cram)